web-admin/src/features/alerts/listing/AlertsTable.svelte

```diff
@@ -11,12 +11,17 @@
   import AlertsTableEmpty from "./AlertsTableEmpty.svelte";
   import AlertsTableHeader from "./AlertsTableHeader.svelte";
   import NoAlertsCTA from "./NoAlertsCTA.svelte";
+  import { isLoadingWithTimeout } from "@rilldata/web-common/features/entity-management/spinner-utils";
 
   export let organization: string;
   export let project: string;
 
   $: alerts = useAlerts($runtime.instanceId);
 
+  const loadingWithTimeout = isLoadingWithTimeout();
+  $: loadingWithTimeout.setLoading($alerts.isLoading);
+  $: loadingAlerts = $loadingWithTimeout;
```

Comment:
Thoughts on updating the `isLoadingWithTimeout()` function to make this a one-liner?

Reply:
sure, I can do that. This helps us remove the boilerplate-y code.

`$: loading = isLoadingWithTimeout(isLoading)` is pretty straight forward.

Reply:
See: https://github.com/rilldata/rill/pull/5351#issuecomment-2266283116